importers/base.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *args, **kwargs):
     self.collection_id = kwargs.get('collection_id', '0')
     self.logfile = kwargs.get('logfile')
     if self.logfile:
-      logging.basicConfig(filename=self.logfile)
+      logging.basicConfig(filename=self.logfile, level=logging.DEBUG)
```

Comment:
Might be nice to have to explicitly opt into logging. Previously if no logfile was set logging was skipped. Now that there's a default value for that there should be a way to omit logging.

Reply:
I implemented a "--quiet" or "-q" parameter that suppresses the logging. But I'm wondering if this is the right way to do it:

`
self.quiet = kwargs.get('quiet', False)
if self.logfile and not self.quiet:
      logging.basicConfig(filename=os.path.join(LOG_DIR, self.logfile), level=logging.DEBUG)
`  
I forgot to test it. What will happen  to all the calls to logging if it isn't initialized here? Will they all throw errors? Do I need to add/update conditionals to all of those? I suspect I do.

Reply:
`logging` will by default print to console: https://docs.python.org/3/howto/logging.html

Reply:
Ok I reworked the `_log(self, level, msg)` function and put this at the beginning of it:
```
    if self.quiet:
      return
```
So that no console commands or logging calls are made if the quite attribute is chosen. I just now posted it.

Reply:
👍 thank you this is better!